### ISSUE TITLE
[a11y] Disabled Icons should be gray

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1430]
**Describe the bug**
You cannot actually add Child Nodes to a Geography form but they display anyway

**To Reproduce**
1. Go to a Geography form
2. Try to add child nodes
3. See error

![image](https://user-images.githubusercontent.com/37256050/167216293-7351e502-7fbf-4235-88ec-aaac8e1b380a.png)

Windows 10, Edge

**Comment by maxxxxxdlp** [comment]
Fixed the bug with icons being colored (disabled icons should be gray)
I didn't remove them completely as icons can be disabled for different reasons, and those icons help visually distinguish subview from formtable from panel from separator

**Comment by grantfitzsimmons** [comment]
Should they be visible? Are there cases where you can add child nodes? Even when looking at a record from a tree I was unable to add them (Geography)

**Comment by maxxxxxdlp** [comment]
by "as icons can be disabled for different reasons" I meant subview icons, in general, can be disabled for several reason, not specifically Geography subview
For example, user may be in read only mode, or lacking permissions or subview is displaying a -to-one relationship and one model is already present

**Comment by grantfitzsimmons** [comment]
Sorry, I meant should child nodes be visible for this type of table? They do not work for me
